webserver/__init__.py

```diff
@@ -25,13 +24,13 @@ def create_app_flaskgroup(script_info):
 def load_config(app):
     """Load configuration file for specified Flask app"""
     config_file = os.path.join(os.path.dirname(os.path.realpath(__file__)), "..", "config.py")
-    for _ in range(CONSUL_CONFIG_FILE_RETRY_COUNT):
-        if not os.path.exists(config_file):
-            time.sleep(1)
+    if deploy_env:
+        for _ in range(CONSUL_CONFIG_FILE_RETRY_COUNT):
+            if not os.path.exists(config_file):
+                time.sleep(1)
 
     if not os.path.exists(config_file):
         print("No config file generated. Retried %d times, exiting." % CONSUL_CONFIG_FILE_RETRY_COUNT)
```

Comment:
This sentence is not strictly true in development any more. Do we care?

Reply:
Perhaps we could fix it by indenting one level?

Reply:
on it.